### ISSUE TITLE
PersistentBottomNavBar/iOS

### ISSUE THREAD

**Issue by Neoncat-OG** [kametepe/BlackHole#5]
Hello, I have forked your repo and I am trying to build the iOS version but I couldn't find an archive of the original PersistentBottomNavBar. If you know of any way to get a working version please let me know. Thanks in advance.

**Comment by AhmedJu5002** [comment]
Hey but for now please fix all the problem this app has .... The whole app is broken... Saavn , Yt-music and YouTube non of them is working... No music can be playing when searched... Please help...

**Comment by Neoncat-OG** [comment]
Hello, this repo fixes the yt and ytmusic issues and Saavn seems to be working fine. I think you are the only one having issues with the entire app. You could try uninstalling and then reinstalling it.

**Comment by AhmedJu5002** [comment]
From where ?? I will reinstall it from where?? The version of the website is not working.. and the link @kaptepe gave me also isn't working... I have reinstalled it several times... Ok you give me the source or the apk file... Then I will reinstall it...

**Comment by AhmedJu5002** [comment]
Not only me all my friends are also facing same issues..Okk you please give me the file to install

**Comment by Neoncat-OG** [comment]
Download this zip https://github.com/kametepe/BlackHole/releases/download/1.15.12-release/blackhole-release-v1.15.12.zip and then install the apk matching your device's architecture inside the zip.

**Comment by AhmedJu5002** [comment]
Yes I downloaded the same file .. and installed on my device and the result is same... Not only my device but also all of my friends' device 😔😔

**Comment by Neoncat-OG** [comment]
The app seems to have broken completely in the past weeks. If I had some of the original code I could help fix it but the dev removed all of his contact information.

**Comment by AhmedJu5002** [comment]
Exactly...umm.. isn't the Ankit sangwan's the original code... And doesn't the start worrier repo have all the original code??

**Comment by AhmedJu5002** [comment]
Hey here is the original repo .. won't you find what you need here??
https://github.com/StarsWarrior/BlackHole

**Comment by Neoncat-OG** [comment]
That repo is a copy of the original but the repo requires dependencies from other repos that the original creator had. This being the repo that I need https://github.com/Sangwan5688/PersistentBottomNavBar.

**Comment by AhmedJu5002** [comment]
Well that means the app is died now??
Well can you tell me that how blackhole fetch music from Yt-music and YouTube in 320 kbps?? When YouTube or Yt-music don't provide this higher quality even for their premium user ...

**Comment by Neoncat-OG** [comment]
I can try to take a closer look into the code and maybe if me and a few others patch up and fill in missing code it could work.
